src/docs/uxd/getStarted/DOC.md

```diff
@@ -1,3 +1,136 @@
+
+## Considerations for designing your app
+
+Before you begin designing, meet with your platform product manager and platform UX designer to discuss design strategies for your app, such as where in the information architecture your capability lives:
+
+Considerations
+Design possibilities
+Does your app relate to existing applications? E.g. Would the same person that uses your app use other apps already on the platform?
+
+Is it a distinct set of capabilities unlike other bundles already on the platform?
+
+Will the capability will be sold/marketed alone or as part of something else?
+It could be an app added to an existing bundle, or
+it could be a new bundle created for a new set of capabilities, or
+Something else - the app might not fit into the current IA or navigation, and might be accessed in some different way. We might even change the IA if needed.
+
+Is it a service that will add capabilities/enhance other apps, and not need to stand alone?
+If yes, then it probably isn’t an app and instead will add attributes or dashboard content or table content to existing apps.
+Is the app targeted for non-customers, like Red Hat solutions architects/consultants working with customers?
+If yes, It may not need to have a presence on the customer-facing landing page or in the left menu, but is accessible through a deep link from another application.
+Other -- something we haven’t considered yet...
+
+This meeting should also involve a discussion about the supporting capabilities available to your content through the platform.
+
+### Platform capabilities
+
+The platform provides several shared capabilities to the apps that join cloud.redhat.com. We strongly recommend that you familiarize yourself with Cloud Services and its existing apps to get the most from the descriptions below.
+
+![Platform capabilities table](https://github.com/RedHatInsights/insights-frontend-storybook/blob/master/src/docs/uxd/img/capabilitiesTable.png?raw=true)
+
+### Capabilities the Platform does not yet provide
+
+* Notification drawer
+* Event log files (customer accessible)
+* Audit log
+* Global (shared among apps) settings. In the future,  global settings customers will access settings from a menu in the masthead. These will likely include RBAC, sources, tagging, and reporting.
+
+If a capability that you need that don’t exist yet, let the platform product manager and your app product manager know. Product Manager: Ian Hands; Platform design IXD: Kendra Marchant. How the platform prioritizes what shared capabilities are available to apps is based on resources and app needs. Your app team may need to design the function if the platform team cannot do it in the timeframe your app requires it.
+
+### Patterns and demos we’re using in apps and the platform already
+
+This list reflects PatternFly4 components we currently use in the production version of Cloud Services and its apps (as of May 6, 2019); we will likely use additional components as the products mature.
```

Comment:
Do we really want to keep a running list of PF components we use? This is going to get out of date really fast.